packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -538,14 +537,17 @@ class SystemChrome {
   ///
   /// {@tool sample}
   /// The following example creates a widget that changes the status bar color
-  /// to a random value on Android.
+  /// to a random value on Android using an `AnnotatedRegion<SystemUiOverlayStyle>`.
```

Comment:
Is this Android specific?

Reply:
Good catch, I forgot to remove this mention which existed before.